src/components/SessionExpiry.jsx

```diff
@@ -18,21 +18,17 @@ const SessionExpiry = () => {
 
   const autoLogOut = () => {
     const countDown = () => {
-      if(isReset){
-        setTimeLeft(600)
-        setIsReset(!isReset)
-      } else {
-        setTimeLeft(timeLeft-.1)
-      }
-    }
-    if(isLoggedIn && timeLeft > 0){
+      isReset ? setTimeLeft(600) && setIsReset(!isReset) : setTimeLeft(timeLeft-.1)
+    }    
+    if(isLoggedIn && (timeLeft > 0)){
       setTimeout(() => {
         countDown();
       },100)
     } else if(isLoggedIn && timeLeft <= 0){
       setIsDismissed(false)
       setIsLoggedIn(false)
       setTimeLeft(0)
+      window.localStorage.clear()
```

Comment:
@andyderek there's a key that we will be storing, which is all that we should clear. look at Search.jsx and ping/dm @ksr89 ; he will tell you how to make the change in that file so that we can simply store the query results instead in a key in localStorage. Then we can simply do `window.localStorage.remove(thatKey)` over here instead;

@ksr89 make sure to write here your suggestions.